STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/src/az_ulib_registry.c

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#include "_az_ulib_registry_interface.h"
+#include "az_ulib_ipc_api.h"
+#include "az_ulib_ipc_interface.h"
+#include "az_ulib_registry_api.h"
+#include "az_ulib_result.h"
+#include "stm32l475_flash_driver.h"
+#include <azure/core/internal/az_precondition_internal.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <string.h>
+
+#define MAX_AZ_ULIB_REGISTRY_ENTRIES 10
+
+static az_registry_block registry_editor[MAX_AZ_ULIB_REGISTRY_ENTRIES];
```

Comment:
Please point it to the flash, no need to have a copy in the RAM. You can define it as 
```
__attribute__((section(".regdata"))) static az_registry_block registry_editor[MAX_AZ_ULIB_REGISTRY_ENTRIES];
```

same for the registry buffer
```
__attribute__((section(".regdata"))) static uint8_t registry_buffer[MAX_AZ_ULIB_REGISTRY_BUFFER];
```

Reply:
discussed offline #WontFix